hoomd/write/table.py

```diff
@@ -205,15 +205,28 @@ def __init__(self,
                  output=output,
                  logger=logger))
         self._param_dict = param_dict
-
         # internal variables that are not part of the state.
+
+        # Generate list of current invalid categories
+        _invalid_inputs = logger.categories & self._invalid_logger_categories
+
         # Ensure that only scalar and potentially string are set for the logger
-        if (LoggerCategories.scalar not in logger.categories
-                or logger.categories & self._invalid_logger_categories
-                !=  # noqa: W504 (yapf formats this incorrectly
-                LoggerCategories.NONE):
+        # If logger.categories == none, pass automatically
```

Comment:
This comment just repeats the next line. Clarification comments like this are good when the next line(s) is/are difficult to parse or call some obscure function.
```suggestion
```